cli/ballerina-cli/src/main/resources/cli-help/internal-error-message.txt

```diff
@@ -1,9 +1,8 @@
 ballerina: Oh no, something really went wrong. Bad. Sad.
 
-There should be a file named "ballerina-internal.log" in the current directory.
 If you are able to share with us the code that broke Ballerina then
 we would REALLY appreciate if you would report this to us:
 go to https://github.com/ballerina-platform/ballerina-lang/issues and
-create a bug report with both this log file and the sample code.
+create a bug report with both this log and the sample code.
 
 We thank you for helping make us better dancers.
```

Comment:
We once got rid of this sentence about helping us be better dancers. But looks like it's back somehow 🤔 Shall we change this as well? What we agreed upon was something like the following: https://github.com/ballerina-platform/ballerina-lang/issues/8286#issuecomment-527356610

Reply:
Fixed with e0be966